### PR TITLE
`deer`: update metadata

### PR DESCRIPTION
**Pull request by indietyp**
## 🌟 What is the purpose of this PR?

Update the `README.md` and `Cargo.toml` to prepare for the initial reservation of crate names.

### CODE REVIEW

**Review by thehabbos007 — APPROVED** [review]
Awesome, thanks! Looks good from my side. David should approve legal stuff before we merge :)

File: libs/deer/desert/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # deer-desert
 
-desert is the the internal only deserialization testing framework used throughout the integration tests and should never
+desert is the internal only deserialization testing framework used throughout the integration tests and should never
```
Comment (TimDiekmann) [inline libs/deer/desert/README.md:3]:
```suggestion
`desert` is the internal only deserialization testing framework used throughout the integration tests and should never
```

File: libs/deer/README.md [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 `deer` is an **experimental** backend-agnostic deserialization framework for Rust, featuring meaningful error messages and context (utilizing [`error-stack`](https://crates.io/crates/error-stack)) and a fail-slow behavior by default.
 
+## ⚠️ Disclaimer ⚠️
+
+This crate does not ship with any functionality and is only a name reservation to stop potential name squatting. In the future the actual crate will be published under this name, for the current (incomplete) implementation of the crate please visit the linked repository.
+
```
Comment (TimDiekmann) [inline libs/deer/README.md:10]: I can add this section when publishing the crates, we don't need it in the repository I guess
```suggestion
```

File: libs/deer/json/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# deer-json
+
+`deer` is an **experimental** backend-agnostic deserialization framework for Rust, featuring meaningful error messages and context (utilizing [`error-stack`](https://crates.io/crates/error-stack)) and a fail-slow behavior by default.
+
+`deer-json` is an implementation of the `deer` deserializer for the JSON data format.
+
+## ⚠️ Disclaimer ⚠️
+
+This crate does not ship with any functionality and is only a name reservation to stop potential name squatting. In the future the actual crate will be published under this name, for the current (incomplete) implementation of the crate please visit the linked repository.
+
```
Comment (TimDiekmann) [inline libs/deer/json/README.md:12]:
```suggestion
```

File: libs/deer/macros/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+# deer-macros
+
+`deer` is an **experimental** backend-agnostic deserialization framework for Rust, featuring meaningful error messages and context (utilizing [`error-stack`](https://crates.io/crates/error-stack)) and a fail-slow behavior by default.
+
+`deer-macros` is a collection of derive macros to automatically implement `Deserialize` for types.
+
+## ⚠️ Disclaimer ⚠️
+
+This crate does not ship with any functionality and is only a name reservation to stop potential name squatting. In the future the actual crate will be published under this name, for the current (incomplete) implementation of the crate please visit the linked repository.
+
```
Comment (TimDiekmann) [inline libs/deer/macros/README.md:12]:
```suggestion
```

**Review by TimDiekmann — APPROVED** [review]

**Review by TimDiekmann — APPROVED** [review]

**Review by nonparibus — APPROVED** [review]

**Pull request merged**